website/static/images/xtable-hero.svg

```diff
@@ -0,0 +1,147 @@
+<svg width="736" height="710" viewBox="0 0 736 710" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Please include the [ASF source header](https://www.apache.org/legal/src-headers.html#headers) in all .svg files

Reply:
Thanks for showing me, I have added this in all the .svg files